framework/wfa/ucc-dummy.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import socket
+import argparse
+import sys
+import threading
+import binascii
+
+def handle(host, port, respDict):
+    id = threading.get_ident()
+    print("{}: Listening on {}:{}".format(id, host, port))
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind((host, port))
+        s.listen()
+        conn, addr = s.accept()
+        with conn:
+            print("{}: {}@{} Connected by {}".format(id, host, port, addr))

```

Comment:
I think its better to keep it as is, so it supports even python2.7. But good to know this is possible with new versions of python.

Reply:
Yes, f-strings are great.

Things will be even better in Python 3.8, where you can write:
```
while data := conn.recv(1024):
```
instead of the `while True`. Python is still adding interesting features.